core/src/main/kotlin/no/ntnu/beardblaster/screen/MenuScreen.kt

```diff
@@ -116,6 +113,6 @@ class MenuScreen(
 
     override fun dispose() {
         super.dispose()
-        UserData.instance.deleteObserver(this)
+        /*UserData.instance.deleteObserver(this)*/
```

Comment:
Should this be commented out when merging to main?

Reply:
Yep, pushed a fix